src/patternfly/components/Table/table.scss

```diff
@@ -118,6 +118,27 @@
   --#{$table}__action--PaddingInlineStart: var(--pf-t--global--spacer--sm);
   --#{$table}__action--PaddingInlineEnd: var(--pf-t--global--spacer--sm);
 
+  // * Table expandable row
+  --#{$table}__expandable-row--TransitionDuration--display: var(--#{$table}__expandable-row--TransitionDuration--fade);
```

Comment:
I'm not seeing where this is used.

Reply:
Thanks! I realized I didn't need that and forgot to delete.